3rd_party/3rd_party.sh

```diff
@@ -192,13 +209,7 @@ case `uname` in
                 if [ $? -eq 0 ] ; then
                     echo "Set RPATH in $FILE"
                 else
-                    # Set RPATH for 3rd party libraries that reference other libraries we ship
-                    ldd $FILE | grep /usr/local/lib >/dev/null 2>&1 && patchelf --set-rpath '$ORIGIN/.' $FILE
```

Comment:
I removed this bit because it hasn't been doing anything since we switched from storing the 3rd party libraries we build from `/usr/local/lib` to `/usr/local/gcc75/lib`.  This came up in this PR because `ldd` only works on the native architecture.  But the line is completely unnecessary now as Boost must have changed the way they link related libraries since we first added this line.